view/theme/frio/templates/admin/logs/view.tpl

```diff
@@ -0,0 +1,136 @@
+<div id="adminpage">
+	<h1>{{$title}} - {{$page}}</h1>
+
+	<h3>{{$logname}}</h3>
+	{{if $error }}
+		<div id="admin-error-message-wrapper" class="alert alert-warning">
+			<p>{{$error nofilter}}</p>
+		</div>
+	{{else}}
+		<form method="get" class="row">
+			<div class="col-xs-10">
+				<div class="form-group form-group-search">
+					<input accesskey="s" id="nav-search-input-field" class="form-control form-search"
+						type="text" name="q" data-toggle="tooltip" title="Search in logs"
+						placeholder="Search" value="{{$q}}">
+					<button class="btn btn-default btn-sm form-button-search"
+						type="submit">Search</button>
+				</div>
+			</div>
+			<div class="xol-xs-2">
+				<a href="/admin/logs/view" class="btn btn-default">Show all</a>
+			</div>
+		</form>
+
+		<table class="table table-hover">
+			<thead>
+				<tr>
+					<th>Date</th>
+					<th class="dropdown">
+						<a class="dropdown-toggle text-nowrap" type="button" id="level" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+							Level {{if $filters.level}}({{$filters.level}}){{/if}}<span class="caret"></span>
+						</a>
+						<ul class="dropdown-menu" aria-labelledby="level">
+							{{foreach $filtersvalues.level as $v }}
+								<li {{if $filters.level == $v}}class="active"{{/if}}>
+									<a href="/admin/logs/view?level={{$v}}" data-filter="level" data-filter-value="{{$v}}">
+										{{if $v == ""}}ALL{{/if}}{{$v}}
+									</a>
+								</li>
+							{{/foreach}}
+						</ul>
+					</th>
+					<th class="dropdown">
+						<a class="dropdown-toggle text-nowrap" type="button" id="context" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+							Context {{if $filters.context}}({{$filters.context}}){{/if}}<span class="caret"></span>
+						</a>
+						<ul class="dropdown-menu" aria-labelledby="context">
+							{{foreach $filtersvalues.context as $v }}
+								<li {{if $filters.context == $v}}class="active"{{/if}}>
+									<a href="/admin/logs/view?context={{$v}}" data-filter="context" data-filter-value="{{$v}}">
+										{{if $v == ""}}ALL{{/if}}{{$v}}
+									</a>
+								</li>
+							{{/foreach}}
+						</ul>
+					</th>
+					<th>Message</th>
+				</tr>
+			</thead>
+			<tbody>
+				{{foreach $data as $row}}
+				<tr id="ev-{{$row->id}}" class="log-event"
```

Comment:
Please add a visual and text cues that this element can be clicked, through a cursor change and an `alt` attribute.